HELM.md

```diff
@@ -58,7 +49,7 @@ Clone the faas-netes repo including the helm chart:
 
 ```
 $ git clone https://github.com/openfaas/faas-netes && \
-  cd faas-netes
+  cd faas-netes/chart
```

Comment:
This was originally called `chart` but it broke the helm linter. This is why it's called `openfaas` instead... thoughts?

Reply:
strange, never had issues putting charts to `charts` folder